{
	"schema_version": "1.6.0",
	"id": "GHSA-2mw7-wggm-m6w3",
	"modified": "2023-11-08T04:00:06.901736Z",
	"published": "2018-11-21T22:22:46Z",
	"aliases": [
		"CVE-2018-19183"
	],
	"summary": "Denial of Service in ethereumjs-vm",
	"details": "ethereumjs-vm 2.4.0 allows attackers to cause a denial of service (vm.runCode failure and REVERT) via a \"code: Buffer.from(my_code, 'hex')\" attribute.",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "npm",
				"name": "ethereumjs-vm",
				"purl": "pkg:npm/ethereumjs-vm"
			},
			"ranges": [
				{
					"type": "SEMVER",
					"events": [
						{
							"introduced": "0"
						},
						{
							"last_affected": "2.4.0"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://github.com/github/advisory-database/blob/main/advisories/github-reviewed/2018/11/GHSA-2mw7-wggm-m6w3/GHSA-2mw7-wggm-m6w3.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2018-19183"
		},
		{
			"type": "WEB",
			"url": "https://github.com/ethereumjs/ethereumjs-vm/issues/386"
		},
		{
			"type": "ADVISORY",
			"url": "https://github.com/advisories/GHSA-2mw7-wggm-m6w3"
		},
		{
			"type": "PACKAGE",
			"url": "https://github.com/ethereumjs/ethereumjs-vm"
		}
	],
	"database_specific": {
		"cwe_ids": [
			"CWE-119"
		],
		"github_reviewed": true,
		"github_reviewed_at": "2020-06-16T20:52:41Z",
		"nvd_published_at": null,
		"severity": "HIGH"
	}
}
